CMake/utils/kwiver-utils-targets.cmake

```diff
@@ -126,10 +148,14 @@ endfunction()
 #-
 function(kwiver_add_executable name)
   add_executable(${name} ${ARGN})
+
+  _kwiver_check_and_set_library_dir()
+  _kwiver_validate_library_dir_value()
+
   set_target_properties(${name}
     PROPERTIES
       RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin"
-      INSTALL_RPATH "\$ORIGIN/../lib:\$ORIGIN/"
+      INSTALL_RPATH "\$ORIGIN/../${library_dir}:\$ORIGIN/"
```

Comment:
https://github.com/Kitware/kwiver/pull/867/files#diff-16539d90ca4b83ae40120635dc2d7966R264 is defined with respect to CMAKE_INSTALL_PREFIX so I would expect that the file would be present in library_dir. Although thinking about this made me realize that this solution is not complete. 
1. If the library_dir has more than 1 directory in it eg. lib/foo then the relative traversing from origin would lead get you outside foo where library_dir is not a valid path.
2. The second problem deals with the intent of the RPATH, if the intent was to point to a default directory containing the libraries then solving the first problem won't fix the issue as the RPATH would point to library_dir not the default directory.